src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerPanel.tsx

```diff
@@ -166,7 +170,7 @@ export const TemplatesCatalogDrawerPanel: FC<TemplatesCatalogDrawerPanelProps> =
                       <VirtualMachineDescriptionItem
                         bodyContent={
                           <CPUDescription
-                            cpu={vmObject?.spec?.template?.spec?.domain?.cpu}
+                            cpu={useCPUData(updatedVM)}
```

Comment:
please save as a const and use it, doing the hook directly is not something I see in our codebase

Reply:
What exactly is problematic about it? We should have better reasons of doing something than just "not seeing something in our codebase".

Reply:
i understand what you are saying, but this is much less readable than save the return data as const and placing it as cpu={cpu}

Reply:
This is kinda tricky because under `vmObject?.spec?.template?.spec?.domain?.cpu` we have cpu data, which is an object containing number of sockets, cores, threads, then we have vcpu count, which is just a number, calculated one. Sometimes we use this, sometimes we need that number. We should distinguish between those. Using a variable called just `cpu` does not say enough, nothing about if it represents a number - vcpu count, or an object containing cpu values (sockets, cores, threads).